src/components/latestIteam.js

```diff
@@ -1,22 +1,25 @@
 import React from 'react'
 import Card from './Card'
 import {products} from '../utils/productData'
-function latestIteams() {
+function latestIteams(countHandler) {
   return (
     <section className='latest-container' >
     <h3>Latest Products</h3>
      <div className="item-container">
      {
-      products?.map((product)=>
+      products?.map((product,i)=>
```

Comment:
this is nice, for an even better approach you can first check if the length of products array is more than 0 or not